src/mpid/ch4/netmod/ofi/ofi_am_impl.h

```diff
@@ -293,7 +293,7 @@ static inline int MPIDI_OFI_am_isend_long(int rank,
     index =
         MPIDI_OFI_index_allocator_alloc(MPIDI_OFI_COMM(MPIR_Process.comm_world).rma_id_allocator,
                                         MPL_MEM_RMA);
-    MPIR_Assert((int) index < MPIDI_Global.max_huge_rmas);
+    MPIR_Assert(index < MPIDI_Global.max_huge_rmas);
```

Comment:
Hmm...this was added here (82b0daf95e2b790f10007cf04e37546264fc1cce) when it shouldn't have been. Not sure why I had it in there.